axolotl-web/src/pages/Contacts.vue

```diff
@@ -37,20 +37,20 @@
               @click="contactClick(c)"
             >
               <div class="badge-name">
-                {{ c.Name[1] ? c.Name[0] + c.Name[1] : c.Name[0] }}
+                <img
+                  class="avatar-img"
+                  :src="'http://localhost:9080/avatars?e164=' + c.Tel"
```

Comment:
Avatar images are stored under the telephone number (_c.Tel)_, is that correct? Why not under the ID?